subprojects/core/src/main/java/org/gradle/util/VersionNumber.java

```diff
@@ -109,7 +109,7 @@ public VersionNumber getBaseVersion() {
 
     @Override
     public int compareTo(VersionNumber other) {
-        logDeprecation(8);
```

Comment:
Maybe it would be better to leave this code as commented line, so it can be easily reverted back later?

Reply:
It's not really an effort to put it back even when completely missing, so better it's more useful to not leave "garbage" lying around.